update_ruleset

```diff
@@ -460,13 +460,13 @@ def main():
     status = {'restart_required': False, 'restarted': False, 'success': False, 'msg': ""}
 
     # Previous checks
-    is_wazuh, ossec_version = get_ossec_version()
+    is_wazuh, wazuh_version = get_wazuh_version()
 
     if not is_wazuh:
-        exit(2, "OSSEC {0} detected. This script only supports Wazuh v2.0 or newer.".format(ossec_version))
+        exit(2, "OSSEC {0} detected. This script only supports Wazuh v2.0 or newer.".format(wazuh_version))
```

Comment:
We can replace here **OSSEC** in this message

Reply:
Exactly. It indicates a pre-Wazuh version.